app/scripts/templates/change_password.mustache

```diff
@@ -1,11 +1,15 @@
-<div id="main-content" class="card">
-  <header>
-    <h1 id="fxa-change-password-header">{{#t}}Change password{{/t}}</h1>
-  </header>
+<div id="change-password" class="settings-unit">
+  <div class="error"></div>
+
+    <div class="settings-unit-stub">
+    <header class="settings-unit-summary">
+      <h2 class="settings-unit-title">{{#t}}Password{{/t}}</h2>
+    </header>
+    <button class="settings-button secondary settings-unit-toggle hidden">{{#t}}Change{{/t}}</button>
+  </div>
```

Comment:
This HTML looks like it's indented "weird" (I think the `<div class="settings-unit-stub">` is indented +2 spaces).